avm/ptn/ai-ml/ai-foundry/main.bicep

```diff
@@ -0,0 +1,368 @@
+metadata name = 'ai-foundry'
+metadata description = 'Creates an AI Foundry account and project with Standard Agent Services.'
+
+targetScope = 'resourceGroup'
+
+import { sqlDatabaseType, deploymentsType } from 'modules/customTypes.bicep'
```

Comment:
I'd highly recommend to **not** use a `customTypes.bicep` file at this time as there's no way for us to publish set types to the registry. If you instead specify them directly in this file (alongside the `@export()` annotation), users can actually import the type via `import { sqlDatabaseType, deploymentsType } from 'br/public:avm/ptn/ai-ml-ai-foundry:0.1.0'` and re-use them for the their parameters when using your module. 
I don't think we have a spec in AVM that prevents you from not doing that, so it's more an opinionated piece of advice based on past user expiences 😉